deployments/airflow/Dockerfile

```diff
@@ -1,13 +0,0 @@
-ARG BASE_IMAGE=apache/airflow:2.7.1-python3.10
```

Comment:
With the airflow `deployment` directory being removed, how does the deployment of airflow with our configurations work now?

Reply:
These files were never used to begin with.